src/components/EnvironmentOverride/environmentOverride.scss

```diff
@@ -67,22 +86,19 @@ section.config-map-overrides {
 .environment-override-list {
     height: 64px;
     .tag {
-        background: var(--N100);
+        border: 1px solid var(--N200);
```

Comment:
can we use css formulae class insted of this?

Reply:
Nope. As we already have different iterations of the class named `.tag` at the global level, it's causing the CSS override issue. Once global CSS name standardization changes get merged then we'll be able to remove these & use formulae at different places. 🙂